### PR TITLE
Updating to RC2

### LINKED ISSUE #1: Move to .NET Core RC2
I am porting one of my apps to RC2 and this is a blocker. Are you working on moving this library over to RC2? I can help, but don't want to duplicate effort, plus I am pretty busy at the moment doing the same for NUnit, nunit/nunit#1371

You may want to change the namespace to match the new standard, `Microsoft.AspNetCore.Identity.EntityFramework6`. If so, would it make sense to fork and release a new library? That said, I doubt people will stay on pre-RC2 long, so maybe not worth it.

### PR DESCRIPTION
**Pull request by rprouse**
Fixes #1

### CODE REVIEW

**Comment by rprouse** [comment]
This is ready to merge. All projects are now building targeting RC2, all tests are passing and the sample project is running and identity works.

I am not sure why AppVeyor is failing, I don't have permissions to see your build.

The only other things you/we may want to do are;
- Update the version
- Change the name to AspNetCore to match all other Asp.NET Core packages.

Let me know if you want either done. I can do them in this PR or as separate PRs.

File: NuGet.config [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
```
Comment (rprouse) [inline NuGet.config:4]: This was needed to pull in some of the testing related packages for the test project. The new versions are not on NuGet.org.

File: test/Microsoft.AspNet.Identity.EntityFramework6.Test/IdentityResultAssert.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Identity.Test
```
Comment (rprouse) [inline test/Microsoft.AspNet.Identity.EntityFramework6.Test/IdentityResultAssert.cs:9]: One of the identity test packages was no longer published, so I had to copy in this class and the TestLogger.

File: test/Microsoft.AspNet.Identity.EntityFramework6.Test/project.json [hunk 3]
```diff
@@ -2,35 +2,41 @@
...
 
-  "compilationOptions": {
-    "warningsAsErrors": true
+  "buildOptions": {
+    "warningsAsErrors": true,
+    "copyToOutput": {
+      "include": [ "config.json" ]
+    }
   },
+  "testRunner": "xunit",
```
Comment (rprouse) [inline test/Microsoft.AspNet.Identity.EntityFramework6.Test/project.json:12]: You can now run tests in the Visual Studio Test Explorer :+1:

**Comment by SharePointRadi** [comment]
Hey Rob, great stuff we will start taking a look at this. AppVeyor will need to be updated so it runs the CI build with RC2.

**Comment by rprouse** [comment]
Would you like me to add an appveyor.yml script that gets this building and producing artifacts? I just did the same for `dotnet-test-nunit`, so it should be fairly similar.

**Comment by SharePointRadi** [comment]
That would be great! I could also give you access to our appveyor account?

**Comment by rprouse** [comment]
Whichever you prefer. The yml worked fine for dotnet-test-nunit.

**Comment by rprouse** [comment]
I have pushed an update that will hopefully get AppVeyor building and producing artifacts. It is injecting the version. Branches and PR's will produce a  nuget package with a prerelease version of -PR-**\* or -Branch-Name, master will produce NuGets that you can release. Assume I didn't make a mistake of course :smile: 

Would you mind going into AppVeyor and giving the NuGet feed a reasonable name by deleting the hash at the end and then post it here? Then I can test the nuget packages that are produced and can use early versions to get my work back on track.

**Comment by rprouse** [comment]
AppVeyor is building and succeeding as far as I can tell,

image

Oddly, GitHub is reporting it as a failure for me. Are you seeing the same?

**Comment by rprouse** [comment]
Latest push is showing as a successful build, so must have been transient.

Green is good, test away :smile:

**Comment by rprouse** [comment]
I am closing this pull request to get it off my pull request list. I will keep the fork/branch around in case you ever want to reopen this PR and merge.

**Pull request closed without merging**